R/textmodel_wordfish.R

```diff
@@ -215,13 +255,23 @@ textmodel_wordfish.dfm <- function(x, dir = c(1, 2),
         psi = as.numeric(result$psi),
         alpha = as.numeric(result$alpha),
         phi = as.numeric(result$phi),
-        se.theta = as.numeric(result$thetaSE) ,
+        zeta = as.numeric(result$zeta),
+        theta2 = as.numeric(result$theta2),
```

Comment:
It probably makes more sense for the 2D case to return theta as an ndoc x 2 matrix instead of as two vectors of length ndoc. Same for the other parameters and SEs.